fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -570,6 +570,10 @@ public String logKey() {
         ONLINE_INDEX_BUILDER_RECORDS_SCANNED("number of records scanned by online index build", false),
         /** The number of records indexed by {@link OnlineIndexer}. */
         ONLINE_INDEX_BUILDER_RECORDS_INDEXED("number of records indexed by online index build", false),
+        /** The number of indexer range termination by scan limit {@link OnlineIndexer}. */
```

Comment:
Perhaps something more like:

```suggestion
        /** The number of {@link OnlineIndexer} range scans terminated after hitting the scan limit. */
```

Reply:
Oh (already visited this item..). How about combining your/my suggesting to: 
        /** The number of {@link OnlineIndexer} records scanned in a single iteration/transaction. */
        /** The number of {@link OnlineIndexer} bytes to write in a single iteration/transaction. */

Reply:
How about 

```
        /** The number of {@link OnlineIndexer} records scanned in a single iteration/transaction. */
        /** The number of {@link OnlineIndexer} bytes to write in a single iteration/transaction. */
```

Reply:
It's probably worth consolidating the discussion here to a single thread. I'll respond in: https://github.com/FoundationDB/fdb-record-layer/pull/999#discussion_r458965386